CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ Unreleased section to make new releases easy.
 
 ## [Unreleased]
 
-Initial release
+### Added
+
```

Comment:
Should we include current version number to group the changes to it?

Reply:
You'll notice its groups under [Unreleased] as I've not actually rolled the next release yet. I didn't want to cause confusion by referencing a release that hadn't been made yet.